web_services/POST/article.json

```diff
@@ -0,0 +1,29 @@
+{
+  "request": {
+    "title": "POST article",
+    "description": "post an Article",
+    "url": "http://localhost/api/article",
+    "method": "POST",
+    "data": {
+      "authorId": "Number",
+      "type": "String",
+      "isCompanyPost": "Boolean",
+      "company":{
+        "companyId": "Number",
+        "isPrior": "Boolean"
+      },
+      "privacy": "String",
+      "text": "String",
+      "hashtags": ["String"],
+      "mentions": ["Number"],
```

Comment:
I think mentions is an array of user-ids. so it should be array of String after changing all ids to String.